terraform-plans/configs/dcgm-snap_main.tfvars

```diff
@@ -10,7 +10,9 @@ templates = {
   check = {
     source      = "./templates/github/snap_check.yaml.tftpl"
     destination = ".github/workflows/check.yaml"
-    vars        = {}
+    vars        = {
+      runs_on = "[[ubuntu-22.04], [Ubuntu_ARM64_4C_16G_01]]",
```

Comment:
It's hard to see if this works without merging; could you test this by opening a draft PR to dcgm-snap with the changes to verify that the workflow passes? :)

Reply:
That was a good suggestion. I had no idea how many things I had to fix until is finally working :grimacing: 

Take a look at https://github.com/canonical/dcgm-snap/pull/15

Reply:
Wow!  Nice work :grin: I vote we merge https://github.com/canonical/dcgm-snap/pull/15 now that it's passing CI; then when we merge this, we can verify that it's produced the same result easily. :)

Reply:
Now that is merged, I'm solving this thread. Thanks Sam!